corehq/apps/sms/migrations/0021_add_keyword.py

```diff
@@ -41,15 +41,6 @@ class Migration(migrations.Migration):
                 ('keyword', models.ForeignKey(to='sms.Keyword', on_delete=models.CASCADE)),
             ],
         ),
-        migrations.CreateModel(
-            name='SQLICDSBackend',
-            fields=[
-            ],
-            options={
-                'proxy': True,
-            },
-            bases=('sms.sqlsmsbackend',),
-        ),
```

Comment:
Instead of adding a new migration to remove `SQLICDSBackend` thought of just not creating this model at all. this would however leave the table added on other environments. So should we actually delete the table instead?

Reply:
You should just add the migration to remove it. At a later point in time we could squash the migrations which would remove references to it.

Reply:
okay, updating.